src/taskEvents/utils.js

```diff
@@ -0,0 +1,124 @@
+const BASE_URL = 'https://api.realdevsquad.com';
+
+function createElement({ type, attributes = {}, innerText }) {
+  const element = document.createElement(type);
+  Object.keys(attributes).forEach((item) => {
+    element.setAttribute(item, attributes[item]);
+  });
+  element.textContent = innerText;
+  return element;
+}
+
+function addLoader(container) {
+  const loader = createElement({
+    type: 'div',
+    attributes: { class: 'loader' },
+  });
+  const loaderP = createElement({
+    type: 'p',
+    attributes: { class: 'loaderP' },
+    innerText: 'Loading...',
+  });
+  loader.appendChild(loaderP);
+  container.appendChild(loader);
+}
+
+function removeLoader() {
+  document.querySelector('.loader').remove();
+}
+
+function addErrorMessage(container) {
+  const errorDiv = createElement({
+    type: 'div',
+    attributes: { class: 'error-div' },
+  });
+  const errorMsg = createElement({
+    type: 'p',
+    attributes: { class: 'error-message' },
+    innerText: 'Something went wrong!',
+  });
+  errorDiv.appendChild(errorMsg);
+  container.appendChild(errorDiv);
+}
+
+async function getTaskLogs(username) {
+  let url;
+  if (username) {
+    url = `${BASE_URL}/logs/tasks?meta.username=${username}`;
```

Comment:
We can use the ternary operator here
const url = username ? `${BASE_URL}/logs/tasks?meta.username=${username}` : `${BASE_URL}/logs/tasks`